sdks/python/apache_beam/ml/inference/vertex_ai_inference.py

```diff
@@ -256,3 +257,46 @@ def validate_inference_args(self, inference_args: Optional[Dict[str, Any]]):
 
   def batch_elements_kwargs(self) -> Mapping[str, Any]:
     return self._batching_kwargs
+
+
+class VertexAITritonModelHandler(ModelHandler):
```

Comment:
this class should be instrumented with the client-side throttling code like the `VertexAIModelHandlerJSON` class.